Sescription: Fix a regression that made ITK test suite fail
 We should not check for presence of Pixel Data at this point. This broke
 a test in ITK test suite.
Author: Mathieu Malaterre <mathieu.malaterre@gmail.com>
Origin: https://sourceforge.net/p/gdcm/gdcm/ci/11c5f4e8a6e8bc6368bdd04f1711f3be10c29a22/
Forwarded: no
Last-Update: 2015-12-21

--- a/Source/DataStructureAndEncodingDefinition/gdcmMediaStorage.cxx
+++ b/Source/DataStructureAndEncodingDefinition/gdcmMediaStorage.cxx
@@ -484,7 +484,9 @@
 bool MediaStorage::SetFromModality(DataSet const &ds)
 {
   // Ok let's try againg with little luck it contains a pixel data...
-  if( ds.FindDataElement( Tag(0x7fe0,0x0010) ) )
+  // technically GDCM is called with a template DataSet before Pixel Data
+  // is even set, so do not check for presence of this attribute at this point
+  //if( ds.FindDataElement( Tag(0x7fe0,0x0010) ) )
     {
     // Pixel Data found !
     // Attempt to recover from the modality (0008,0060):
@@ -512,7 +514,7 @@
       }
     return true;
     }
-  return false;
+  //return false;
 }
 
 bool MediaStorage::SetFromFile(File const &file)
